modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -337,4 +470,39 @@ protected function assertElementsTexts(array $expected, array $elements): void {
     ));
   }
 
+  /**
+   * Create a list page node with filters configured.
+   *
+   * @param string $bundle
+   *   Nodes of this bundle will be listed.
+   * @param array $exposed_filters
+   *   Facet machine names linked to the bundle's facet source.
+   *
+   * @return \Drupal\node\NodeInterface
+   *   The list page node created.
+   */
+  protected function createListPage(string $bundle, array $exposed_filters): NodeInterface {
+    $list_page = Node::create([
+      'type' => 'oe_list_page',
+      'title' => 'Results',
+    ]);
+
+    /** @var \Drupal\emr\Entity\EntityMetaInterface $list_page_entity_meta */
+    $list_page_entity_meta = $list_page->get('emr_entity_metas')->getEntityMeta('oe_list_page');
+    /** @var \Drupal\oe_list_pages\ListPageWrapper $list_page_entity_meta_wrapper */
+    $list_page_entity_meta_wrapper = $list_page_entity_meta->getWrapper();
+    $list_page_entity_meta_wrapper->setSource('node', $bundle);
+    $list_page_entity_meta_wrapper->setConfiguration([
+      'override_exposed_filters' => 1,
+      'exposed_filters' => $exposed_filters,
```

Comment:
I like this method. But let's simplify the `$exposed_filters` parameter.
Let it expect a simple array of strings, and then here we can call `array_combine($exposed_filters, $exposed_filters)`.
```suggestion
      'exposed_filters' => array_combine($exposed_filters, $exposed_filters),
```
See other comments above.

Reply:
If we will ever want to do something like 
```
      'oelp_oe_sc_event__location' => 'oelp_oe_sc_event__location',
      'oelp_oe_sc_event__title' => 0,
      'oelp_oe_sc_event__oe_sc_event_dates' => 'oelp_oe_sc_event__oe_sc_event_dates',
```
we wont be able to, but for our test seems like an edge case, at least at this point, so ok for now, lets see what the future brings.

Reply:
Or we could skip that item in that case.
Although, the reason to include it would be to verify that all the options exist.

But for this we could also pass this instead:
```php
      'oelp_oe_sc_event__location' => TRUE,
      'oelp_oe_sc_event__title' => FALSE,
      'oelp_oe_sc_event__oe_sc_event_dates' => TRUE,
```